src/components/ChatEntry.js

```diff
@@ -1,22 +1,30 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp  from './TimeStamp';
 
 const ChatEntry = (props) => {
+
+  const heartColor = props.liked ? '❤️' : '🤍';
+  const bubbleDirection = props.id % 2 === 0 ? 'chat-entry remote' : 'chat-entry local';
```

Comment:
Very clever solution to this problem! Making use of a pattern you see in your data and implementing a solution is what makes great coders!